packages/chaire-lib-backend/src/utils/processManagers/TrRoutingProcessManager.ts

```diff
@@ -81,10 +81,11 @@ const startTrRoutingProcess = async (
     return processStatus;
 };
 
+const getCacheAllScenariosFromConfig = () => serverConfig.getTrRoutingConfig('single').cacheAllScenarios;
```

Comment:
Do we still need this function?

Reply:
yeah, the function contains the 'single' instance definition, so it gets less error-prone and we are sure that all function that use it will use the exact same config. But it's debatable. It's less necessary now.

Reply:
Worst case, a comment with a WARNING to not use with the batch path